core/templates/dev/head/pages/base.html

```diff
@@ -119,6 +119,10 @@
       {% endif %}
     </script>
 
+    <!-- Note to developers: This script needs to be loaded before external
```

Comment:
Actually, thinking about it -- not sure you need a separate script. Why not just put it inline here? One less fetch.